source/group_eq_bot/driver/local/text/base.py

```diff
@@ -0,0 +1,12 @@
+from pathlib import Path
+
+
+class BaseTextReader:
+
+    @staticmethod
+    def open(path_to_read: Path) -> str:
```

Comment:
Suggest to rename function as `read`, main purpose of the Reader is to `read`

Reply:
Applied